slack_sdk/models/metadata/__init__.py

```diff
@@ -28,3 +29,1207 @@ def __str__(self):
 
     def __repr__(self):
         return self.__str__()
+
+
+## Work object entity metadata
+
+
+class EntityType(StrEnum):
```

Comment:
[StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) was introduced in python `3.11` so I don't think this will work when using the project in python `3.7`